### PR TITLE
Changes for deployment of rpm and dpkg.

### PR DESCRIPTION
**Pull request by joachimmetz**
[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/155)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,6 +116,7 @@
         os.path.join('tools', 'stats.py'),
         os.path.join('tools', 'validator.py'),
     ],
+    cmdclass={'bdist_rpm': BdistRPMCommand}
```
Comment (joachimmetz) [inline setup.py]: Note to self, add missing ,

File: config/dpkg/control [hunk 2]
```diff
@@ -1,14 +1,23 @@
-Source: python-artifacts
+Source: artifacts
 Section: python
 Priority: extra
 Maintainer: Forensic artifacts <forensicartifacts@googlegroups.com>
-Build-Depends: debhelper (>= 7.0.0), python, python-setuptools
-Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 7), python-all (>= 2.7~), python-setuptools, python3-all (>= 3.2~), python3-setuptools
```
Comment (destijl) [inline config/dpkg/control]: Is there any point packaging the python with the artifacts? I imagine anyone installing this package only wants the yaml?

Comment (joachimmetz) [inline config/dpkg/control]: The idea is to use it with plaso, no need to re-write the registry and reading functionality. 
https://github.com/joachimmetz/plaso/blob/artifacts/plaso/engine/engine.py#L94

**Comment by joachimmetz** [comment]
@destijl friendly ping

**Comment by destijl** [comment]
LGTM

**Pull request merged**